[2024-12-16T18:31:38.606+0000] {processor.py:157} INFO - Started process (PID=30) to work on /opt/airflow/dags/scripts/kafka_streaming2.py
[2024-12-16T18:31:38.609+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/scripts/kafka_streaming2.py for tasks to queue
[2024-12-16T18:31:38.610+0000] {logging_mixin.py:150} INFO - [2024-12-16T18:31:38.609+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/scripts/kafka_streaming2.py
[2024-12-16T18:31:38.625+0000] {logging_mixin.py:150} INFO - [2024-12-16T18:31:38.623+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/scripts/kafka_streaming2.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/scripts/kafka_streaming2.py", line 1, in <module>
    from confluent_kafka import Producer
ModuleNotFoundError: No module named 'confluent_kafka'
[2024-12-16T18:31:38.627+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/scripts/kafka_streaming2.py
[2024-12-16T18:31:38.674+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/scripts/kafka_streaming2.py took 0.077 seconds
[2024-12-16T18:32:09.290+0000] {processor.py:157} INFO - Started process (PID=40) to work on /opt/airflow/dags/scripts/kafka_streaming2.py
[2024-12-16T18:32:09.291+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/scripts/kafka_streaming2.py for tasks to queue
[2024-12-16T18:32:09.292+0000] {logging_mixin.py:150} INFO - [2024-12-16T18:32:09.292+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/scripts/kafka_streaming2.py
[2024-12-16T18:32:09.302+0000] {logging_mixin.py:150} INFO - [2024-12-16T18:32:09.298+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/scripts/kafka_streaming2.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/scripts/kafka_streaming2.py", line 1, in <module>
    from confluent_kafka import Producer
ModuleNotFoundError: No module named 'confluent_kafka'
[2024-12-16T18:32:09.302+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/scripts/kafka_streaming2.py
[2024-12-16T18:32:09.325+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/scripts/kafka_streaming2.py took 0.036 seconds
[2024-12-16T18:33:23.197+0000] {processor.py:157} INFO - Started process (PID=29) to work on /opt/airflow/dags/scripts/kafka_streaming2.py
[2024-12-16T18:33:23.201+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/scripts/kafka_streaming2.py for tasks to queue
[2024-12-16T18:33:23.204+0000] {logging_mixin.py:150} INFO - [2024-12-16T18:33:23.204+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/scripts/kafka_streaming2.py
[2024-12-16T18:33:23.229+0000] {logging_mixin.py:150} INFO - [2024-12-16T18:33:23.224+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/scripts/kafka_streaming2.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/scripts/kafka_streaming2.py", line 1, in <module>
    from confluent_kafka import Producer
ModuleNotFoundError: No module named 'confluent_kafka'
[2024-12-16T18:33:23.233+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/scripts/kafka_streaming2.py
[2024-12-16T18:33:23.383+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/scripts/kafka_streaming2.py took 0.193 seconds
[2024-12-16T18:33:53.876+0000] {processor.py:157} INFO - Started process (PID=39) to work on /opt/airflow/dags/scripts/kafka_streaming2.py
[2024-12-16T18:33:53.878+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/scripts/kafka_streaming2.py for tasks to queue
[2024-12-16T18:33:53.879+0000] {logging_mixin.py:150} INFO - [2024-12-16T18:33:53.878+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/scripts/kafka_streaming2.py
[2024-12-16T18:33:53.884+0000] {logging_mixin.py:150} INFO - [2024-12-16T18:33:53.883+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/scripts/kafka_streaming2.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/scripts/kafka_streaming2.py", line 1, in <module>
    from confluent_kafka import Producer
ModuleNotFoundError: No module named 'confluent_kafka'
[2024-12-16T18:33:53.885+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/scripts/kafka_streaming2.py
[2024-12-16T18:33:53.897+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/scripts/kafka_streaming2.py took 0.023 seconds
[2024-12-16T18:34:24.446+0000] {processor.py:157} INFO - Started process (PID=49) to work on /opt/airflow/dags/scripts/kafka_streaming2.py
[2024-12-16T18:34:24.447+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/scripts/kafka_streaming2.py for tasks to queue
[2024-12-16T18:34:24.448+0000] {logging_mixin.py:150} INFO - [2024-12-16T18:34:24.448+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/scripts/kafka_streaming2.py
[2024-12-16T18:34:24.451+0000] {logging_mixin.py:150} INFO - [2024-12-16T18:34:24.451+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/scripts/kafka_streaming2.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/scripts/kafka_streaming2.py", line 1, in <module>
    from confluent_kafka import Producer
ModuleNotFoundError: No module named 'confluent_kafka'
[2024-12-16T18:34:24.452+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/scripts/kafka_streaming2.py
[2024-12-16T18:34:24.464+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/scripts/kafka_streaming2.py took 0.019 seconds
[2024-12-16T18:34:55.198+0000] {processor.py:157} INFO - Started process (PID=59) to work on /opt/airflow/dags/scripts/kafka_streaming2.py
[2024-12-16T18:34:55.198+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/scripts/kafka_streaming2.py for tasks to queue
[2024-12-16T18:34:55.199+0000] {logging_mixin.py:150} INFO - [2024-12-16T18:34:55.199+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/scripts/kafka_streaming2.py
[2024-12-16T18:34:55.204+0000] {logging_mixin.py:150} INFO - [2024-12-16T18:34:55.202+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/scripts/kafka_streaming2.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/scripts/kafka_streaming2.py", line 1, in <module>
    from confluent_kafka import Producer
ModuleNotFoundError: No module named 'confluent_kafka'
[2024-12-16T18:34:55.205+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/scripts/kafka_streaming2.py
[2024-12-16T18:34:55.218+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/scripts/kafka_streaming2.py took 0.022 seconds
[2024-12-16T18:35:25.915+0000] {processor.py:157} INFO - Started process (PID=69) to work on /opt/airflow/dags/scripts/kafka_streaming2.py
[2024-12-16T18:35:25.916+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/scripts/kafka_streaming2.py for tasks to queue
[2024-12-16T18:35:25.917+0000] {logging_mixin.py:150} INFO - [2024-12-16T18:35:25.917+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/scripts/kafka_streaming2.py
[2024-12-16T18:35:25.921+0000] {logging_mixin.py:150} INFO - [2024-12-16T18:35:25.920+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/scripts/kafka_streaming2.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/scripts/kafka_streaming2.py", line 1, in <module>
    from confluent_kafka import Producer
ModuleNotFoundError: No module named 'confluent_kafka'
[2024-12-16T18:35:25.921+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/scripts/kafka_streaming2.py
[2024-12-16T18:35:25.934+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/scripts/kafka_streaming2.py took 0.020 seconds
[2024-12-16T18:35:56.682+0000] {processor.py:157} INFO - Started process (PID=79) to work on /opt/airflow/dags/scripts/kafka_streaming2.py
[2024-12-16T18:35:56.682+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/scripts/kafka_streaming2.py for tasks to queue
[2024-12-16T18:35:56.683+0000] {logging_mixin.py:150} INFO - [2024-12-16T18:35:56.683+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/scripts/kafka_streaming2.py
[2024-12-16T18:35:56.687+0000] {logging_mixin.py:150} INFO - [2024-12-16T18:35:56.686+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/scripts/kafka_streaming2.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/scripts/kafka_streaming2.py", line 1, in <module>
    from confluent_kafka import Producer
ModuleNotFoundError: No module named 'confluent_kafka'
[2024-12-16T18:35:56.687+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/scripts/kafka_streaming2.py
[2024-12-16T18:35:56.700+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/scripts/kafka_streaming2.py took 0.021 seconds
[2024-12-16T18:36:27.559+0000] {processor.py:157} INFO - Started process (PID=89) to work on /opt/airflow/dags/scripts/kafka_streaming2.py
[2024-12-16T18:36:27.560+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/scripts/kafka_streaming2.py for tasks to queue
[2024-12-16T18:36:27.561+0000] {logging_mixin.py:150} INFO - [2024-12-16T18:36:27.561+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/scripts/kafka_streaming2.py
[2024-12-16T18:36:27.564+0000] {logging_mixin.py:150} INFO - [2024-12-16T18:36:27.564+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/scripts/kafka_streaming2.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/scripts/kafka_streaming2.py", line 1, in <module>
    from confluent_kafka import Producer
ModuleNotFoundError: No module named 'confluent_kafka'
[2024-12-16T18:36:27.564+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/scripts/kafka_streaming2.py
[2024-12-16T18:36:27.577+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/scripts/kafka_streaming2.py took 0.019 seconds
[2024-12-16T18:36:58.271+0000] {processor.py:157} INFO - Started process (PID=99) to work on /opt/airflow/dags/scripts/kafka_streaming2.py
[2024-12-16T18:36:58.273+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/scripts/kafka_streaming2.py for tasks to queue
[2024-12-16T18:36:58.273+0000] {logging_mixin.py:150} INFO - [2024-12-16T18:36:58.273+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/scripts/kafka_streaming2.py
[2024-12-16T18:36:58.277+0000] {logging_mixin.py:150} INFO - [2024-12-16T18:36:58.276+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/scripts/kafka_streaming2.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/scripts/kafka_streaming2.py", line 1, in <module>
    from confluent_kafka import Producer
ModuleNotFoundError: No module named 'confluent_kafka'
[2024-12-16T18:36:58.277+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/scripts/kafka_streaming2.py
[2024-12-16T18:36:58.289+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/scripts/kafka_streaming2.py took 0.020 seconds
[2024-12-16T18:37:29.067+0000] {processor.py:157} INFO - Started process (PID=109) to work on /opt/airflow/dags/scripts/kafka_streaming2.py
[2024-12-16T18:37:29.067+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/scripts/kafka_streaming2.py for tasks to queue
[2024-12-16T18:37:29.068+0000] {logging_mixin.py:150} INFO - [2024-12-16T18:37:29.068+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/scripts/kafka_streaming2.py
[2024-12-16T18:37:29.072+0000] {logging_mixin.py:150} INFO - [2024-12-16T18:37:29.071+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/scripts/kafka_streaming2.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/scripts/kafka_streaming2.py", line 1, in <module>
    from confluent_kafka import Producer
ModuleNotFoundError: No module named 'confluent_kafka'
[2024-12-16T18:37:29.072+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/scripts/kafka_streaming2.py
[2024-12-16T18:37:29.090+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/scripts/kafka_streaming2.py took 0.025 seconds
[2024-12-16T18:38:00.118+0000] {processor.py:157} INFO - Started process (PID=119) to work on /opt/airflow/dags/scripts/kafka_streaming2.py
[2024-12-16T18:38:00.118+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/scripts/kafka_streaming2.py for tasks to queue
[2024-12-16T18:38:00.119+0000] {logging_mixin.py:150} INFO - [2024-12-16T18:38:00.119+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/scripts/kafka_streaming2.py
[2024-12-16T18:38:00.123+0000] {logging_mixin.py:150} INFO - [2024-12-16T18:38:00.122+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/scripts/kafka_streaming2.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/scripts/kafka_streaming2.py", line 1, in <module>
    from confluent_kafka import Producer
ModuleNotFoundError: No module named 'confluent_kafka'
[2024-12-16T18:38:00.123+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/scripts/kafka_streaming2.py
[2024-12-16T18:38:00.140+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/scripts/kafka_streaming2.py took 0.025 seconds
[2024-12-16T18:38:30.966+0000] {processor.py:157} INFO - Started process (PID=129) to work on /opt/airflow/dags/scripts/kafka_streaming2.py
[2024-12-16T18:38:30.967+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/scripts/kafka_streaming2.py for tasks to queue
[2024-12-16T18:38:30.968+0000] {logging_mixin.py:150} INFO - [2024-12-16T18:38:30.967+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/scripts/kafka_streaming2.py
[2024-12-16T18:38:30.972+0000] {logging_mixin.py:150} INFO - [2024-12-16T18:38:30.970+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/scripts/kafka_streaming2.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/scripts/kafka_streaming2.py", line 1, in <module>
    from confluent_kafka import Producer
ModuleNotFoundError: No module named 'confluent_kafka'
[2024-12-16T18:38:30.972+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/scripts/kafka_streaming2.py
[2024-12-16T18:38:30.985+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/scripts/kafka_streaming2.py took 0.021 seconds
[2024-12-16T18:39:01.750+0000] {processor.py:157} INFO - Started process (PID=139) to work on /opt/airflow/dags/scripts/kafka_streaming2.py
[2024-12-16T18:39:01.750+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/scripts/kafka_streaming2.py for tasks to queue
[2024-12-16T18:39:01.751+0000] {logging_mixin.py:150} INFO - [2024-12-16T18:39:01.751+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/scripts/kafka_streaming2.py
[2024-12-16T18:39:01.755+0000] {logging_mixin.py:150} INFO - [2024-12-16T18:39:01.754+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/scripts/kafka_streaming2.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/scripts/kafka_streaming2.py", line 1, in <module>
    from confluent_kafka import Producer
ModuleNotFoundError: No module named 'confluent_kafka'
[2024-12-16T18:39:01.755+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/scripts/kafka_streaming2.py
[2024-12-16T18:39:01.768+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/scripts/kafka_streaming2.py took 0.020 seconds
[2024-12-16T18:39:32.519+0000] {processor.py:157} INFO - Started process (PID=149) to work on /opt/airflow/dags/scripts/kafka_streaming2.py
[2024-12-16T18:39:32.519+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/scripts/kafka_streaming2.py for tasks to queue
[2024-12-16T18:39:32.520+0000] {logging_mixin.py:150} INFO - [2024-12-16T18:39:32.520+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/scripts/kafka_streaming2.py
[2024-12-16T18:39:32.524+0000] {logging_mixin.py:150} INFO - [2024-12-16T18:39:32.523+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/scripts/kafka_streaming2.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/scripts/kafka_streaming2.py", line 1, in <module>
    from confluent_kafka import Producer
ModuleNotFoundError: No module named 'confluent_kafka'
[2024-12-16T18:39:32.525+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/scripts/kafka_streaming2.py
[2024-12-16T18:39:32.548+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/scripts/kafka_streaming2.py took 0.031 seconds
[2024-12-16T18:40:03.260+0000] {processor.py:157} INFO - Started process (PID=159) to work on /opt/airflow/dags/scripts/kafka_streaming2.py
[2024-12-16T18:40:03.261+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/scripts/kafka_streaming2.py for tasks to queue
[2024-12-16T18:40:03.262+0000] {logging_mixin.py:150} INFO - [2024-12-16T18:40:03.262+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/scripts/kafka_streaming2.py
[2024-12-16T18:40:03.266+0000] {logging_mixin.py:150} INFO - [2024-12-16T18:40:03.265+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/scripts/kafka_streaming2.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/scripts/kafka_streaming2.py", line 1, in <module>
    from confluent_kafka import Producer
ModuleNotFoundError: No module named 'confluent_kafka'
[2024-12-16T18:40:03.267+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/scripts/kafka_streaming2.py
[2024-12-16T18:40:03.295+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/scripts/kafka_streaming2.py took 0.036 seconds
[2024-12-16T18:43:35.269+0000] {processor.py:157} INFO - Started process (PID=30) to work on /opt/airflow/dags/scripts/kafka_streaming2.py
[2024-12-16T18:43:35.270+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/scripts/kafka_streaming2.py for tasks to queue
[2024-12-16T18:43:35.271+0000] {logging_mixin.py:150} INFO - [2024-12-16T18:43:35.271+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/scripts/kafka_streaming2.py
[2024-12-16T18:43:35.279+0000] {logging_mixin.py:150} INFO - [2024-12-16T18:43:35.277+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/scripts/kafka_streaming2.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/scripts/kafka_streaming2.py", line 1, in <module>
    from confluent_kafka import Producer
ModuleNotFoundError: No module named 'confluent_kafka'
[2024-12-16T18:43:35.279+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/scripts/kafka_streaming2.py
[2024-12-16T18:43:35.316+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/scripts/kafka_streaming2.py took 0.051 seconds
[2024-12-16T18:44:05.645+0000] {processor.py:157} INFO - Started process (PID=40) to work on /opt/airflow/dags/scripts/kafka_streaming2.py
[2024-12-16T18:44:05.646+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/scripts/kafka_streaming2.py for tasks to queue
[2024-12-16T18:44:05.647+0000] {logging_mixin.py:150} INFO - [2024-12-16T18:44:05.647+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/scripts/kafka_streaming2.py
[2024-12-16T18:44:05.651+0000] {logging_mixin.py:150} INFO - [2024-12-16T18:44:05.650+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/scripts/kafka_streaming2.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/scripts/kafka_streaming2.py", line 1, in <module>
    from confluent_kafka import Producer
ModuleNotFoundError: No module named 'confluent_kafka'
[2024-12-16T18:44:05.651+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/scripts/kafka_streaming2.py
[2024-12-16T18:44:05.670+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/scripts/kafka_streaming2.py took 0.027 seconds
[2024-12-16T18:44:36.255+0000] {processor.py:157} INFO - Started process (PID=50) to work on /opt/airflow/dags/scripts/kafka_streaming2.py
[2024-12-16T18:44:36.255+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/scripts/kafka_streaming2.py for tasks to queue
[2024-12-16T18:44:36.256+0000] {logging_mixin.py:150} INFO - [2024-12-16T18:44:36.256+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/scripts/kafka_streaming2.py
[2024-12-16T18:44:36.260+0000] {logging_mixin.py:150} INFO - [2024-12-16T18:44:36.259+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/scripts/kafka_streaming2.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/scripts/kafka_streaming2.py", line 1, in <module>
    from confluent_kafka import Producer
ModuleNotFoundError: No module named 'confluent_kafka'
[2024-12-16T18:44:36.261+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/scripts/kafka_streaming2.py
[2024-12-16T18:44:36.273+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/scripts/kafka_streaming2.py took 0.019 seconds
[2024-12-16T18:45:53.614+0000] {processor.py:157} INFO - Started process (PID=30) to work on /opt/airflow/dags/scripts/kafka_streaming2.py
[2024-12-16T18:45:53.621+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/scripts/kafka_streaming2.py for tasks to queue
[2024-12-16T18:45:53.622+0000] {logging_mixin.py:150} INFO - [2024-12-16T18:45:53.622+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/scripts/kafka_streaming2.py
[2024-12-16T18:45:53.642+0000] {logging_mixin.py:150} INFO - [2024-12-16T18:45:53.637+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/scripts/kafka_streaming2.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/scripts/kafka_streaming2.py", line 1, in <module>
    from confluent_kafka import Producer
ModuleNotFoundError: No module named 'confluent_kafka'
[2024-12-16T18:45:53.642+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/scripts/kafka_streaming2.py
[2024-12-16T18:45:53.690+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/scripts/kafka_streaming2.py took 0.104 seconds
[2024-12-16T18:46:35.762+0000] {processor.py:157} INFO - Started process (PID=30) to work on /opt/airflow/dags/scripts/kafka_streaming2.py
[2024-12-16T18:46:35.763+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/scripts/kafka_streaming2.py for tasks to queue
[2024-12-16T18:46:35.764+0000] {logging_mixin.py:150} INFO - [2024-12-16T18:46:35.764+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/scripts/kafka_streaming2.py
[2024-12-16T18:46:35.771+0000] {logging_mixin.py:150} INFO - [2024-12-16T18:46:35.768+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/scripts/kafka_streaming2.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/scripts/kafka_streaming2.py", line 1, in <module>
    from confluent_kafka import Producer
ModuleNotFoundError: No module named 'confluent_kafka'
[2024-12-16T18:46:35.772+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/scripts/kafka_streaming2.py
[2024-12-16T18:46:35.811+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/scripts/kafka_streaming2.py took 0.050 seconds
[2024-12-16T18:47:06.038+0000] {processor.py:157} INFO - Started process (PID=40) to work on /opt/airflow/dags/scripts/kafka_streaming2.py
[2024-12-16T18:47:06.039+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/scripts/kafka_streaming2.py for tasks to queue
[2024-12-16T18:47:06.040+0000] {logging_mixin.py:150} INFO - [2024-12-16T18:47:06.039+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/scripts/kafka_streaming2.py
[2024-12-16T18:47:06.046+0000] {logging_mixin.py:150} INFO - [2024-12-16T18:47:06.044+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/scripts/kafka_streaming2.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/scripts/kafka_streaming2.py", line 1, in <module>
    from confluent_kafka import Producer
ModuleNotFoundError: No module named 'confluent_kafka'
[2024-12-16T18:47:06.047+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/scripts/kafka_streaming2.py
[2024-12-16T18:47:06.064+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/scripts/kafka_streaming2.py took 0.028 seconds
[2024-12-16T18:47:36.688+0000] {processor.py:157} INFO - Started process (PID=50) to work on /opt/airflow/dags/scripts/kafka_streaming2.py
[2024-12-16T18:47:36.689+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/scripts/kafka_streaming2.py for tasks to queue
[2024-12-16T18:47:36.690+0000] {logging_mixin.py:150} INFO - [2024-12-16T18:47:36.690+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/scripts/kafka_streaming2.py
[2024-12-16T18:47:36.695+0000] {logging_mixin.py:150} INFO - [2024-12-16T18:47:36.694+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/scripts/kafka_streaming2.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/scripts/kafka_streaming2.py", line 1, in <module>
    from confluent_kafka import Producer
ModuleNotFoundError: No module named 'confluent_kafka'
[2024-12-16T18:47:36.696+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/scripts/kafka_streaming2.py
[2024-12-16T18:47:36.710+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/scripts/kafka_streaming2.py took 0.024 seconds
[2024-12-16T18:48:07.679+0000] {processor.py:157} INFO - Started process (PID=60) to work on /opt/airflow/dags/scripts/kafka_streaming2.py
[2024-12-16T18:48:07.680+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/scripts/kafka_streaming2.py for tasks to queue
[2024-12-16T18:48:07.680+0000] {logging_mixin.py:150} INFO - [2024-12-16T18:48:07.680+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/scripts/kafka_streaming2.py
[2024-12-16T18:48:07.684+0000] {logging_mixin.py:150} INFO - [2024-12-16T18:48:07.683+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/scripts/kafka_streaming2.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/scripts/kafka_streaming2.py", line 1, in <module>
    from confluent_kafka import Producer
ModuleNotFoundError: No module named 'confluent_kafka'
[2024-12-16T18:48:07.685+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/scripts/kafka_streaming2.py
[2024-12-16T18:48:07.700+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/scripts/kafka_streaming2.py took 0.022 seconds
[2024-12-16T18:48:38.426+0000] {processor.py:157} INFO - Started process (PID=70) to work on /opt/airflow/dags/scripts/kafka_streaming2.py
[2024-12-16T18:48:38.427+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/scripts/kafka_streaming2.py for tasks to queue
[2024-12-16T18:48:38.428+0000] {logging_mixin.py:150} INFO - [2024-12-16T18:48:38.428+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/scripts/kafka_streaming2.py
[2024-12-16T18:48:38.431+0000] {logging_mixin.py:150} INFO - [2024-12-16T18:48:38.431+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/scripts/kafka_streaming2.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/scripts/kafka_streaming2.py", line 1, in <module>
    from confluent_kafka import Producer
ModuleNotFoundError: No module named 'confluent_kafka'
[2024-12-16T18:48:38.432+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/scripts/kafka_streaming2.py
[2024-12-16T18:48:38.444+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/scripts/kafka_streaming2.py took 0.020 seconds
[2024-12-16T18:49:08.934+0000] {processor.py:157} INFO - Started process (PID=80) to work on /opt/airflow/dags/scripts/kafka_streaming2.py
[2024-12-16T18:49:08.935+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/scripts/kafka_streaming2.py for tasks to queue
[2024-12-16T18:49:08.936+0000] {logging_mixin.py:150} INFO - [2024-12-16T18:49:08.935+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/scripts/kafka_streaming2.py
[2024-12-16T18:49:08.943+0000] {logging_mixin.py:150} INFO - [2024-12-16T18:49:08.941+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/scripts/kafka_streaming2.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/scripts/kafka_streaming2.py", line 1, in <module>
    from confluent_kafka import Producer
ModuleNotFoundError: No module named 'confluent_kafka'
[2024-12-16T18:49:08.943+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/scripts/kafka_streaming2.py
[2024-12-16T18:49:08.964+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/scripts/kafka_streaming2.py took 0.032 seconds
[2024-12-16T18:49:39.943+0000] {processor.py:157} INFO - Started process (PID=90) to work on /opt/airflow/dags/scripts/kafka_streaming2.py
[2024-12-16T18:49:39.944+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/scripts/kafka_streaming2.py for tasks to queue
[2024-12-16T18:49:39.944+0000] {logging_mixin.py:150} INFO - [2024-12-16T18:49:39.944+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/scripts/kafka_streaming2.py
[2024-12-16T18:49:39.948+0000] {logging_mixin.py:150} INFO - [2024-12-16T18:49:39.947+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/scripts/kafka_streaming2.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/scripts/kafka_streaming2.py", line 1, in <module>
    from confluent_kafka import Producer
ModuleNotFoundError: No module named 'confluent_kafka'
[2024-12-16T18:49:39.949+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/scripts/kafka_streaming2.py
[2024-12-16T18:49:39.965+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/scripts/kafka_streaming2.py took 0.024 seconds
